### ISSUE TITLE
[BUG] Skript ignores event cancellation.

### ISSUE THREAD

**Issue by JeffreyRCGitHub** [SkriptLang/Skript#5752]
### Skript/Server Version

```markdown
Server Version: git-Paper-448 (MC: 1.19.3) 
Skript Version: 2.6.4 
Installed Skript Addons:
- Skellet v2.0.9 
-skRayFall v1.9.28 
- SkQuery v4.1.9 
- SkBee v2.8.2 
Installed dependencies: 
- Vault v1.7.3-b131 
- WorldGuard v7.0.7+216b061
```

### Bug Description

Skript ignores event cancellation. When a plugin like CommandWhitelist tries to cancel the event it doesn’t work. Source: github.com/YouHaveTrouble/CommandWhitelist/issues/73

### Expected Behavior

It is supposed to cancel the event of the permission message of skript showing.

### Steps to Reproduce

1. Get CommandWhitelist for Bukkit https://www.spigotmc.org/resources/commandwhitelist-spigot-waterfall-velocity.81326/.
2. Put CommandWhitelist's .jar file in the plugins folder.
3. Make this code with the file name test.sk:
```v
command test:
  permission: permission.test
  trigger:
    send "test" to player
```
4. Save your file and do /sk reload test.
5. CommandWhitelist's permission message should pop up but skript's permission message pops up too.

### Agreement

- [X] I have read the guidelines above and affirm I am following them with this report.

**Comment by TheLimeGlass** [comment]
This is because Skript is **NOT** actually make a registered command. Skript just listens within a `on command` event when the phase is said `/test`.

If you wish for Skript to take priority over the plugin in question. Change the event priority of Skript in the config.sk or you can even use the priority pattern after any event with `[with priority (lowest|low|normal|high|highest|monitor)]`

**Comment by JeffreyRCGithub** [comment]
CommandWhitelist (https://www.spigotmc.org/resources/commandwhitelist-spigot-waterfall-velocity.81326/) is for blocking commands and tab completion so people can’t see your plugins with /[TAB] and looking for a name before “:". When CommandWhitelist tries to get rid of the permission message it doesn’t work.

**Comment by sovdeeth** [comment]
![image](https://github.com/SkriptLang/Skript/assets/10354869/0bea36a7-09f7-4054-a942-a06662125dbb)
Yes, Skript ignores all cancellations when listening for commands to run. I don't think it should do this.

**Comment by sovdeeth** [comment]
@JeffreyRCGithub Try this nightly jar for 2.7: https://github.com/SkriptLang/Skript/actions/runs/6066909722
I can make you one for 2.6.4 if you can't test on 2.7 for some reason.

**Comment by UnderscoreTud** [comment]
We usually leave issues open until a version that contains the fix/feature is released.

**Comment by sovdeeth** [comment]
> We usually leave issues open until a version that contains the fix/feature is released.

It should actually have been included in 2.7.1 I believe, it was cherrypicked from the feature branch to the patch branch. So it should be good to close now. I forgot about it in favor of the other issue #5966 fixed.
